app/controllers/manage_evaluators_controller.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+class ManageEvaluatorsController < ApplicationController
+  include ManageEvaluatorsHelper
+
+  before_action :set_challenge
```

Comment:
I got logged out and didn't realize it, when I tried to delete an evaluator it gave me an error message back. I think we need to require logged in user on these controllers.
![Screenshot 2024-11-07 at 4 02 35 PM](https://github.com/user-attachments/assets/b805602a-9dd3-4b50-8ea9-a0d5192c475a)
![Screenshot 2024-11-07 at 4 03 58 PM](https://github.com/user-attachments/assets/59bfb665-ead1-4ec5-9df5-3800d460739b)
We've been using the `before_action :authorize_user` for this elsewhere

Reply:
```suggestion
    before_action -> { authorize_user('challenge_manager') }
    before_action :set_challenge
```